examples/discovery-search-app/src/app.scss

```diff
@@ -116,19 +116,10 @@ $nav-toolbar-height: 42px;
     top: 0;
   }
 
-  .#{$prefix}--document-preview {
```

Comment:
I removed some stylings in the Discovery Search App so that it resembles Tooling more. It had a lot of strange margins and padding.